doc/src/manual/methods.md

```diff
@@ -335,7 +336,20 @@ Here's an example where the method type parameter `T` is used as the type parame
 type `Vector{T}` in the method signature:
 
 ```jldoctest
-julia> myappend(v::Vector{T}, x::T) where {T} = [v..., x]
+julia> function myappend(v::Vector{T}, x::T) where {T}
+           return [v..., x]
+       end
+myappend (generic function with 1 method)
+```
+
+The type parameter `T` in the function's signature ensures that the type of the new `v` is preserved after element `x` is appended to it. 
+The function's signature is preceded by the keyword `where` and is placed immediately after the method arguments. 
+When only one type parameter is used, the curly braces are optional. 
+Parametric method definition also works with the short-form syntax.
+An equivalent one-line definition would be:
+
+```jldoctest
+julia> myappend(v::Vector{T}, x::T) where T = [v..., x]
 myappend (generic function with 1 method)
 
 julia> myappend([1,2,3],4)
```

Comment:
I would move the examples to the first code block.

Reply:
Your suggestions sound good (but not being able to see all the code blocks in this window, I'm a little lost). I'm not sure how to edit from here. In this window this is what I see:

<img width="1424" alt="b" src="https://user-images.githubusercontent.com/6339274/117062169-f9926880-acd7-11eb-92ab-6348e4cef614.png">

If I go back to the main window into the "Commits" tab, this is what I see:

<img width="1425" alt="a" src="https://user-images.githubusercontent.com/6339274/117062253-1af35480-acd8-11eb-9b33-65d2f9531cdb.png">

I just don't know where I'm supposed to click now.

Sorry this is getting tedious here. Feel free to commit the changes you see fit. You guys have managed to help me clarify the main misunderstandings and the parts of the docs look clearer to me now. Thanks for your help.

Reply:
I don’t have write access to the julia repo, so I don’t think it will let me push to your branch. I can try to put all my suggestions into one review, then you should be able to commit them.

Reply:
Sounds good. Please do. Sorry that I'm new to the commit routine. Thanks!

Reply:
I have just figured out I could add you as a "collaborator" to give you write access. I added you and Simeon. Feel free to edit if you have time/desire. Thanks a lot.